cheerio-scraper/INPUT_SCHEMA.json

```diff
@@ -197,6 +197,24 @@
 			"default": {},
 			"prefill": {},
 			"editor": "json"
+		},
+		"datasetName": {
+			"title": "Dataset Name",
+			"type": "string",
+			"description": "Name or ID of the dataset that will be used for storing results.",
```

Comment:
I would add: 

Name or ID of the dataset that will be used for storing results. If left empty, the default dataset of the run will be used.